recipes/qat_distributed.py

```diff
@@ -235,7 +237,22 @@ def setup(self, cfg: DictConfig) -> None:
             else None,
         )
 
+        # initialize loss
         self._loss_fn = config.instantiate(cfg.loss)
+        backend = os.environ.get("TORCH_COMPILE_BACKEND", "inductor")
+        if self._loss_fn.__class__.__name__ == "CEWithChunkedOutputLoss":
+            # set num_output_chunks for model
+            self._model.set_num_output_chunks(self._loss_fn.num_output_chunks)
+            if self._model_compile:
+                log.info("Compiling loss with torch.compile...")
+                self._loss_fn._compute_cross_entropy = torch.compile(
+                    self._loss_fn._compute_cross_entropy, backend=backend
+                )
```

Comment:
Could be good to add a quick comment here around why we compile this method specifically (I know you already explain in detail in the class, but this is more visible)